tests/rules/flowint/test.yaml

```diff
@@ -0,0 +1,17 @@
+requires:
+    min-version: 9.0
+    pcap: false
+
+args:
+    - --engine-analysis
+
+checks:
+- filter:
+    filename: rules.json
+    count: 1
+    match:
+      id: 1
+      lists.postmatch.matches[0].name: "flowint"
+      lists.postmatch.matches[0].flowint.cmd: "unknown"
```

Comment:
I know that this is probably due to the commented out portion of the code that went with this PR, but `unknown` would be incorrect, here.